packages/rum/README.md

```diff
@@ -44,6 +44,8 @@ datadogRum.init({
   - `service`: name of the corresponding service
   - `env`: environment of the service
   - `version`: version of the service
+  - `enableTracing`: enable tracing of xhr and fetch requests
+  - `shouldTraceUrl`: function to determine if an URL should be traced (default: same domain only)
```

Comment:
If this is set, is the same domain still traced?

Reply:
not in the current implementation, it allows customers to exclude some same domain endpoints if they want to

Reply:
That could be confusing if this is used to _add_ domains and then the default is no longer traced.

Reply:
a good doc could address that

Reply:
Something simple with good defaults should not need to be documented.